src/devtools/client/debugger/src/components/QuickOpenModal.tsx

```diff
@@ -80,7 +84,13 @@ interface QOMState {
 export class QuickOpenModal extends Component<PropsFromRedux, QOMState> {
   constructor(props: PropsFromRedux) {
     super(props);
-    this.state = { results: null, selectedIndex: 0 };
+
+    this.state = {
+      results: props.showOnlyOpenSources
+        ? this.formatSources(props.sourcesForTabs, props.tabs)
+        : null,
+      selectedIndex: 0,
+    };
```

Comment:
By default, the normal search shows _nothing_ until you enter query text (but then even if you erase it all, it shows a list of unfiltered results– which is kind of weird). Anyway the new search mode shows _all open files_ by default, and then you can further refine that list with query text if you want.